features/in_onderzoek.feature

```diff
@@ -185,721 +184,939 @@ Functionaliteit: in onderzoek
-      | aanduiding gegevens in onderzoek (83.10) | <waarde>                              |
-      | datum ingang onderzoek (83.20)           | 20220307                              |
+      | gegeven in onderzoek      | waarde | type                   | adresregel1 | adresregel2 | korteNaam | straat | woonplaats | verbl.plaatstype | verbl.adres datum | verbl.adres datumtype | verbl.adres lang |
+      | gemeenteVanInschrijving   | 080910 |                        |             | true        |           |        |            |                  |                   |                       |                  | 
+      | straat                    | 081110 | VerblijfplaatsOnbekend | true        |             | true      | true   |            | true             | 2022-03-07        | Datum                 | 7 maart 2022     | 
+      | naamOpenbareRuimte        | 081115 | VerblijfplaatsOnbekend | true        |             |           | true   |            | true             | 2022-03-07        | Datum                 | 7 maart 2022     | 
+      | huisnummer                | 081120 |                        | true        |             |           |        |            |                  |                   |                       |                  | 
+      | woonplaats                | 081170 | VerblijfplaatsOnbekend |             | true        |           |        | true       | true             | 2022-03-07        | Datum                 | 7 maart 2022     | 
+      | locatiebeschrijving       | 081210 |                        | true        |             |           |        |            |                  |                   |                       |                  | 
+      | adresregel1               | 081330 |                        | true        |             |           |        |            |                  |                   |                       |                  | 
+      | groep gemeente            | 080900 |                        |             | true        |           |        |            |                  |                   |                       |                  | 
+      | groep adres               | 081100 | VerblijfplaatsOnbekend | true        | true        | true      | true   | true       | true             | 2022-03-07        | Datum                 | 7 maart 2022     | 
+      | groep locatiebeschrijving | 081200 |                        | true        |             |           |        |            |                  |                   |                       |                  | 
+      | groep verblijf buitenland | 081300 |                        | true        | true        |           |        |            |                  |                   |                       |                  | 
+      | categorie verblijfplaats  | 080000 | VerblijfplaatsOnbekend | true        | true        | true      | true   | true       | true             | 2022-03-07        | Datum                 | 7 maart 2022     | 
+
```

Comment:
JHe vraagt velden uit de verblijfplaats op dus zal je in elke respons een type van de verblijfplaats terugkrijgen. Nu is di ein regel 1093, 1096, 1098, 1099, 1100, 1102, en 1103 niet geveuld.

Reply:
Volgens mij heb je gelijk. Daarnaast zag ik dat verblijfplaats.inOnderzoek.type hier en daar ten onrechte niet of juist wel geleverd werd en voor verblijfplaats.verblijfadres.inOnderzoek.datumIngangOnderzoek geldt hetzelfde.

Reply:
In 'verblijfplaats.inOnderzoek' moet in sommige situaties (als 'type' inOnderzoek is) ook 'datumIngangOnderzoek' worden gegenereerd.